src/parseline.c

```diff
@@ -846,23 +846,29 @@ void m_nick (char *sender, char *tail)
 
     uptr2 = find_user(newnick);
     if (uptr2) {
-        if (uptr && (find_link2(oldnick,newnick) || find_link2(newnick,oldnick)) && uptr->authed == 1) {
+        if (uptr && (find_link2(oldnick, newnick) || find_link2(newnick, oldnick)) && uptr->authed == 1)
+        {
             uptr->authed = 0;
             uptr->lastseen = time(NULL);
             uptr2->authed = 1;
             SendRaw("SVSMODE %s +r",newnick);
             uptr2->lastseen = time(NULL);
-        } else {
+        }
+        else
```

Comment:
Nit: coding style (the } else { was perfectly fine)

Reply:
Fixed